app/src/main/res/layout/fragment_user.xml

```diff
@@ -1,24 +1,200 @@
+
+    android:id="@+id/parent_scroll"
     android:layout_width="match_parent"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:background="@color/teal_200"
-    android:layout_height="match_parent"
-    tools:context=".UserFragment">
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:background="@color/white"
+        android:layout_height="match_parent"
+        tools:context=".UserFragment">
+
+
+        <LinearLayout
+            android:id="@+id/layout_user"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            android:layout_marginTop="@dimen/big_size"
+            android:layout_marginEnd="@dimen/big_size"
+            android:layout_marginStart="@dimen/big_size"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <com.google.android.material.imageview.ShapeableImageView
+                android:id="@+id/frag_user_user_photo"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="@drawable/image_user"/>
+
+            <com.google.android.material.textview.MaterialTextView
+                android:id="@+id/frag_user_user_name"
+                android:layout_width="0dp"
+                android:text="Juan perez"
+                android:textColor="@color/black"
+                android:textStyle="bold"
+                android:layout_marginStart="@dimen/medium_size"
+                android:textSize="@dimen/big_size_text"
+                android:layout_height="match_parent"
+                android:layout_weight="0.5"/>
+
+            <com.google.android.material.imageview.ShapeableImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:backgroundTint="@color/colorApp"
+                android:background="@drawable/ic_settings"/>
+
+
+        </LinearLayout>
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/frag_user_tv_blanc1"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/small_size"
+            android:layout_marginTop="@dimen/medium_size"
+            android:background="@color/buscar_edit_text"
+            app:layout_constraintTop_toBottomOf="@id/layout_user"
+            app:layout_constraintEnd_toEndOf="@id/layout_user"
+            app:layout_constraintStart_toStartOf="@id/layout_user"/>
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/frag_user_recyclerView"
+            android:layout_width="0dp"
+            android:layout_height="140dp"
```

Comment:
you're already using the dimens file for other specs, please continue with that good practice and avoid hardcoding sizes